conference/accounts.py

```diff
@@ -35,7 +35,7 @@
     "You need to consent to use of your data before we can continue"
 )
 
-EMAIL_VERIFICATION_SUBJECT = "EuroPython2020: Please verify your email"
+EMAIL_VERIFICATION_SUBJECT = "EuroPython 2020: Please verify your email"
```

Comment:
and why not using `settings.CONFERENCE_NAME` ?

Reply:
Yes, that's better. I'll fix those places.